src/fastcs/logging/_logging.py

```diff
@@ -30,8 +31,9 @@ def _configure_logger(
     graylog_env_fields: GraylogEnvFields | None = None,
 ):
     logger.remove()
+    is_pytest_on_windows = "PYTEST_VERSION" in os.environ and os.name == "nt"
     logger.add(
-        sink=StdoutProxy(raw=True),  # type: ignore
+        sink=sys.stdout if is_pytest_on_windows else StdoutProxy(raw=True),  # type: ignore
```

Comment:
Could this be moved to a fixture in the tests to keep the production code unchanged? I think something like this would do it

```python
@pytest.fixture(scope="session", autouse=True)
def patch_stdoutproxy_on_windows(mocker: MockerFixture):
    if os.name == "nt":
        import fastcs.logging._logging as logging_mod

        mocker.patch.object(
            logging_mod,
            "StdoutProxy",
            return_value=sys.stdout,
        )
```